pastas/rfunc.py

```diff
@@ -881,9 +880,18 @@ def step(self, p, dt=1, cutoff=None, maxtmax=None):
         return s
 
 
-class Kleur(RfuncBase):
-    """The function of Kraijenhoff van de Leur, describing the response of a
-    polder domain with length L between two ditches.
+class Kraijenhoff(RfuncBase):
+    """The function of Kraijenhoff van de Leur scaled. The function describes
```

Comment:
Class description should start with a one line description. The rest goes into "Notes" section.